web/src/main/java/org/springframework/security/web/authentication/logout/CompositeLogoutHandler.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.web.authentication.logout;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.security.core.Authentication;
+
+/**
+ * @author Eddú Meléndez
+ */
+public class CompositeLogoutHandler implements LogoutHandler {
+
+	private List<LogoutHandler> logoutHandlers;
+
+	public CompositeLogoutHandler(LogoutHandler... logoutHandlers) {
+		this.logoutHandlers = Arrays.asList(logoutHandlers);
+	}
+
```

Comment:
Please add another constructor that accepts a List. This will allow HttpServlet3RequestFactory to be cleaned up.